src/System.Collections.Immutable/src/Properties/SecurityTransparent.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Security;
+
+[assembly: SecurityTransparent]
```

Comment:
> I don't think I'm comfortable with blindly doing this

Thanks... just to be clear, are you referring to this specific change to immutable collections, or are you referring to my question about doing it elsewhere as well?